lib/index.js

```diff
@@ -3,15 +3,11 @@ import fs from 'fs';
 
 fetch('https://www.reddit.com/r/pics/search.json?q=programming&sort=new')
   .then(res => res.json())
-  .then(res => {
+  .then((res) => {
     const data = JSON.stringify(res);
 
-    fs.writeFile('data.json', data, (err) => {
-      if (err) throw err;
-
-      console.log('Created Successfully');
-    });
+    fs.promises.writeFile('data.json', data)
```

Comment:
Nice work identifying that promises are available in the standard `fs` library!

It's best practice to alias `fs.promises` at import time rather than when using the library.

Example:
```js
// es6 import
import { promises as fs } from 'fs';

// require
const fs = require('fs').promises;

// usage
fs.writeFile();
```

Reply:
Okay, that makes sense.